### PR TITLE
(wip) add informational popover base component and sample

### PR DESCRIPTION
**Pull request by maryhipp**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [x] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [ ] Yes
- [ ] No

## Description
Adds a new common component `IAIInformationPopover` that composes JSX to be rendered within a popover as a tooltip. We were not able to use the `Tooltip` component provided by chakra because you cannot interact with elements within those (at least not that I could get working). 

This just a sample over positive prompt. We need content from @hipsterusername and @Millu before we can roll this out.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

File: invokeai/frontend/web/src/features/informationalPopovers/components/clipSkip.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from 'react';
+import IAIInformationalPopover from '../../../common/components/IAIInformationalPopover';
+import { useTranslation } from 'react-i18next';
+
+export const ClipSkipPopover = (props: PropsWithChildren) => {
+  const { t } = useTranslation();
+
+  return (
+    <IAIInformationalPopover
+      paragraph={t('popovers.clipSkip.paragraph')}
```
Comment (maryhipp) [inline invokeai/frontend/web/src/features/informationalPopovers/components/clipSkip.tsx:10]: You could probably get rid of all of these components if you did something like this:

in the component: 
```
<IAIInformationPopover details="popovers.clipSkip" >
   ...children
</IAIInformationPopover>
```

And then in the IAIInformationPopover, update it to use the paragraph and heading based on the `details` string passed in (assuming you can interpolate the translation key which may not be possible). Then you would only have the one `details` props (and the trigger component, which could always be children) and just deduct the paragraph and heading based on that key.
```
const paragraphKey = `${props.details}.paragraph`
const headerKey = `${props.details}.heading`
if (paragraphKey && and headingKey) {
  ....use those in the JSX
}
```

Comment (maryhipp) [reply invokeai/frontend/web/src/features/informationalPopovers/components/clipSkip.tsx:10]: That would also make it easier to add new ones - just add the object in the translation file, and use it as a `details` prop in the IAIInformationalPopover component around whatever they want to show it

Comment (chainchompa) [reply invokeai/frontend/web/src/features/informationalPopovers/components/clipSkip.tsx:10]: How about also doing a Popovers.json file? Each popover is also gonna need an image, button href/label, and optional placement and idk if we should be putting all that into the translation file.

**Comment by Millu** [comment]
This is sweet! Feel like this will be a big QoL improvement for folks

Experiencing an issue where I'm having to click on a dropdown to select it, then click again to expand the dropdown, but II can't tell if this is due to the popover or something else

https://github.com/invoke-ai/InvokeAI/assets/7254508/d90f0aab-d531-4c1e-bbf3-f8c7a8918f48

**Comment by Millu** [comment]
> Are you having this issue on the main branch as well? I'm not able to replicate it on firefox or chrome

I'm only seeing it on this branch, unfortunately

**Review by hipsterusername — APPROVED** [review]

**Pull request merged**